app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/BaseViewAnnotationActivity.kt

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.maps.testapp.examples.markersandcallouts
+
+import android.view.Menu
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.maps.testapp.R
+import com.mapbox.maps.viewannotation.ViewAnnotationManager
+import com.mapbox.maps.viewannotation.ViewAnnotationUpdateMode
+
+abstract class BaseViewAnnotationActivity : AppCompatActivity() {
```

Comment:
I see the point of introducing this base activity, since we have 3 view annotation related activities, but as examples, maybe each activity should be more self-contained, and it's easier to publish these examples to the `www` later as well.

Reply:
well...
guess you're correct, will revert then